app/services/invoice_custom_sections/select_service.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module InvoiceCustomSections
+  class SelectService < BaseService
+    def initialize(section:, organization:)
```

Comment:
Does this service really need `organization` as input parameter?

`InvoiceCustomSection` has `belongs_to :organization` association and we can relay on it, simplify this service by removing `if organization` checks and also eliminating issue of potential assigning section from one organization to another.

Reply:
yep, we need it: we're going to introduce also selections for customers as well, because an invoice_custom_section can be not selected for the organization, but selected for a customer (when customer does not follow organization selection)